css/view.css

```diff
@@ -3,6 +3,15 @@
   display: none !important;
 }
 
-span.tvm__text:not(:has(span)) {
+/* hides "see how you compare to x applicants" option next to reactivate premium button */
+
+.job-details-jobs-unified-top-card__job-insight--highlight:has(div):has(
+    div
+  ):has(div):has(svg[data-test-icon="lightbulb-medium"]) {
+  display: none !important;
+}
+
+/* hides applicant count at top of page next to company name and location */
+.tvm__text--neutral {
```

Comment:
I refactored the code that hides the applicant count at the top of the page. After examining the CSS more closely I realized that the other two options (company name and location) have a different class name.